src/wp-admin/css/forms.css

```diff
@@ -551,6 +551,17 @@ fieldset label,
 
 .wp-pwd {
 	margin-top: 1em;
+	position: relative;
+}
+
+.wp-pwd button {
+	height: min-content;
+	width: 4.875rem;
```

Comment:
Fixed width buttons can be trouble in other languages.

Spanish overflows a little:
![setup-config in Spanish](https://user-images.githubusercontent.com/17100257/216280497-7ef9fd18-c29c-478a-aaf9-29db0b1c0153.png)

Malayalam overflows a lot:
![setup-config in Malayalam](https://user-images.githubusercontent.com/17100257/216280495-725e144c-a485-4c11-b310-2aa8dc8835ad.png)